internal/api/acrsdk.go

```diff
@@ -94,7 +95,9 @@ func newAcrCLIClientWithBasicAuth(loginURL string, username string, password str
 func newAcrCLIClientWithBearerAuth(loginURL string, refreshToken string) (AcrCLIClient, error) {
 	newAcrCLIClient := newAcrCLIClient(loginURL)
 	ctx := context.Background()
-	accessTokenResponse, err := newAcrCLIClient.AutorestClient.GetAcrAccessToken(ctx, loginURL, "registry:catalog:* repository:*:*", refreshToken)
+	// For ABAC-enabled registries, only request catalog scope initially
+	// Repository-specific scopes will be requested when needed
+	accessTokenResponse, err := newAcrCLIClient.AutorestClient.GetAcrAccessToken(ctx, loginURL, "registry:catalog:*", refreshToken)
```

Comment:
We can't simply remove the repository scope check from `newAcrCLIClientWithBearerAuth`. The new `refreshTokenForRepository` method is only invoked when the existing token has expired. Other files still call this method to obtain the CLI client with the token ([example](https://github.com/Azure/acr-cli/blob/77486acc2efa8893b778f757f34a3325347d4cc8/cmd/acr/manifest.go#L69))